ipi/inputs/motion/motion.py

```diff
@@ -179,11 +179,18 @@ def store(self, motion):
 
         if type(motion) is MultiMotion:
             self.mode.store("multi")
-            self.extra = []
-            for m in motion.mlist:
-                im = InputMotionBase()
-                im.store(m)
-                self.extra.append(("motion", im))
+
+
+            if len(self.extra) != len(motion.mlist):
```

Comment:
@ceriottm : what is this for exactly? Seems unrelated. Is it from another branch?